microprofile.jdt/com.redhat.microprofile.jdt.core/static-properties/mp-lra-metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"properties": [
+		{
+			"type": "java.lang.String",
+			"extensionName": "microprofile-lra",
```

Comment:
I think it should be `microprofile-lra-api` (I mean the artifact id, please use the proper artifact id). I think we have the same problem with other startic json files (we should end extension name with -api, but please check that).

I'm sorry it's me that I suggest you to use that, but I was wrong.

Reply:
Sounds good. Since `-api` is in the artifact name for lra, metrics, openapi and opentracing, my latest push adds the `-api` at the end of the extension names.